frontend/src/app/main/ui/auth/login.cljs

```diff
@@ -170,33 +182,53 @@
   [:div.auth-buttons
    (when (contains? @cf/flags :login-with-google)
      [:a.btn-primary.btn-large.btn-google-auth
-      {:on-click #(login-with-oidc % :google params)}
+      {:tab-index "0"
+       :on-click #(login-with-oidc % :google params)
+       :on-key-down (fn [event]
+                     (when (k/enter? event)
+                       (login-with-oidc event :google params)))}
```

Comment:
Perhaps we could create a `button-link` component, with all this behavior by default